scripts/build/libc/glibc.sh

```diff
@@ -247,6 +247,10 @@ do_libc_backend_once() {
     [ -n "${CT_TOOLCHAIN_BUGURL}" ] && extra_config+=("--with-bugurl=${CT_TOOLCHAIN_BUGURL}")
 
     touch config.cache
+
+    # Hide host C++ binary from configure
+    echo "ac_cv_prog_ac_ct_CXX=" >>config.cache
```

Comment:
Sort of. The generated files are only used during the build of glibc. They're used to override the behaviour of gcc >= 6. The glibc build creates dummy cstdlib and cmake include files if it needs to so that the real files (which use #include_next) won't turn up as a make dependency. This manifestes on non-linux machines as an install failure in the libc step. The relevant comment from glibc is in its Makerules file, line 113.